modules/ballerina-core/src/main/java/org/ballerinalang/model/values/BStringArray.java

```diff
@@ -52,6 +52,7 @@ public String get(long index) {
         return values[(int) index];
     }
 
+    @Deprecated
```

Comment:
@sameera-jayasoma @hasithaa and others. Looks like we can remove this method because the size method has the correct behavior.

Reply:
@pamod @afkham Shall we remove this method. There is no use other than this.

Reply:
@afkham @hasithaa removed the method https://github.com/ballerinalang/ballerina/pull/4398/commits/be302cc44213da28393cdb856a625b63b5f53329

Reply:
Looks good for merging.